.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,16 @@
+<!-- Enter a brief description/summary of your PR here. What does it fix/what does it change/how was it tested (even manually, if necessary)? -->
+## Summary of the Pull Request
+
+<!-- Other than the issue solved, is this relevant to any other issues/existing PRs? --> 
+## References
+
+<!-- Please review the items on the PR checklist before submitting-->
+## PR Checklist
+* [ ] Closes #xxx
+* [ ] CLA signed. If not, go over [here](https://cla.opensource.microsoft.com/microsoft/Terminal) and sign the CLA
+* [ ] Tests added/passed
+* [ ] Requires documentation to be updated
+* [ ] I've discussed this with core contributors already. If not checked, I'm ready to accept this work might be rejected in favor of a different grand plan. Issue number where discussion took place: #___
```

Comment:
This line uses #___ and the one at the top is #xxx.

I don't have a preference other than that they should both be the same placeholder.

Reply:
resolved